docs/spec/openapi.yml

```diff
@@ -0,0 +1,3026 @@
+openapi: "3.0.2"
+info:
+  title: GuideWire insuranceNow cloud API
+  version: 5.0.0
+
+paths:
+  /addresses/countries:
+    get:
+      operationId: getSupportedCountries
+      description: Returns a list of supported countries.
+      parameters:
+        - in: query
+          name: sortType
+          schema:
+            type: string
+            enum: 
+              - asc
+              - desc
+            default: asc
+          description: Indicates the method used to sort the results by name.
+      responses:
+        "200":
+          description: Successful response.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ListCountryResponse"
```

Comment:
Shall we rename this object schema for `ListCountry` or something without `Response`, having `Response` in the name can be misleading to a section with Response in the component response schema. 
Please check other Response schemas.

Reply:
Fixed with [4f822b353b17aa352032637e305d3dbfddc796ab](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/2/commits/4f822b353b17aa352032637e305d3dbfddc796ab)

Reply:
@lnash94 Just out of curiosity, what if we have a request schema as well for the same operation? :) don't we have have to use `[Foo]Request` and `[Foo]Response` prefixes in such scenarios anyway?